pkg/storage/s3/s3_test.go

```diff
@@ -157,18 +166,26 @@ func TestConfigEnv(t *testing.T) {
 	listers := testBuilder.BuildListers()
 
 	d := NewDriver(ctx, config, listers)
+	defer func() {
+		conf, err := d.GetConfig()
```

Comment:
This will create a new config file, why do you need this?

Reply:
This is some remnants from an earlier approach where the S3 driver would keep only one copy of the AWS config/creds file. Removing it now...